experiment/coverage/apicoverage.go

```diff
@@ -137,6 +138,8 @@ func getAPILog(restlog string) apiArray {
 	return parseAPILog(fp)
 }
 
+var reOpenapi = regexp.MustCompile(`({\S+?})`)
```

Comment:
Move back?

Reply:
Yeah, the reOpenapi is used in getTestedAPIs() only.
So I just move it to here for the code readability.
Sorry for mixing it in this PR.